src/components/SummitCard/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+import calendarIcon from '../../../static/img/summit-landing/icons/calendar-icon.svg'
+import locationIcon from '../../../static/img/summit-landing/icons/location-icon.svg'
+
+import './styles.scss'
+
+const SummitCard = ({ background, date, location, notification = false, cardStyles }) => {
+    return (
+        <section className='summit-card-wrapper' style={cardStyles}>
+            <div className="summit-card-image" style={{ backgroundImage: `url(${background}` }}></div>
+            <div className={`summit-card-info ${notification ? '' : 'no-notification'}`}>
+                <div className='summit-card-info-wrapper'>
+                    <span>
+                        <img src={calendarIcon} /> {date}
+                    </span>
+                    <span>
+                        <img src={locationIcon} /> {location}
+                    </span>
+                </div>
+            </div>
+            {notification &&
```

Comment:
@tomrndom 
instead of have a boolean (notification) u could said to pass a a single object , summit
that has
summit = {
date,
location
notification: {
text,
link
}
}
being summit.notification object optional